### PR TITLE
[JENKINS-41511] Don't try to set slaveAgentPort when it is enforced

### PR DESCRIPTION
**Pull request by Vlatombe**
@reviewbybees

### CODE REVIEW

**Comment by amuniz** [comment]
:bee:

**Comment by alvarolobato** [comment]
:bee:

**Comment by Vlatombe** [comment]
@reviewbybees done

File: core/src/main/java/hudson/security/GlobalSecurityConfiguration.java [hunk 1]
```diff
@@ -79,7 +79,7 @@ public int getSlaveAgentPort() {
      * @since 2.24
      * @return true if the slave agent port is enforced on this instance.
      */
-    @Restricted(DoNotUse.class) // only for index.groovy
+    @Restricted(NoExternalUse.class) // only for index.groovy
```
Comment (daniel-beck) [inline core/src/main/java/hudson/security/GlobalSecurityConfiguration.java]: Obsolete comment left.

**Pull request merged**